TempleEight/docs/getting-started.md

```diff
@@ -4,3 +4,400 @@ title: Getting Started
+
+## Running The Example
+
+Now that we have created our example Templefile, save it in a file named `dating-app.temple`, in our `temple-tutorial` directory created earlier.
+
+### Validating the Templefile
+
+The Temple CLI contains a tool for validating your Templefile before generating any code. We can invoke this tool by running:
+
+```
+~/Documents/temple-tutorial ❯❯❯ temple validate dating-app.temple
+Templefile validated correctly
+```
+
+Which means our Templefile is syntactically correct, and we an begin to generate all the code we need.
+
+### Generating Code
+
+At this point, we can begin to generate some code. Running: 
+
+```
+~/Documents/temple-tutorial ❯❯❯ temple generate dating-app.temple
+```
+
+Begins the process. The first thing we'll see is the CLI prompting us to give some further generation details that are specific to the metadata selections made in our Templefile. 
+
+```
+~/Documents/temple-tutorial ❯❯❯ temple generate dating-app.temple
+What should the Go module name be? (expected format "github.com/username/repo")
+```
+
+We need to enter the name of the Golang module we'll be generating into for our project, this can be anything, but standards dictate it should be the github URL of where this project will be hosted. 
+
+For this example enter `github.com/temple/tutorial`. 
+
+```
+~/Documents/temple-tutorial ❯❯❯ temple generate dating-app.temple
+What should the Go module name be? (expected format "github.com/username/repo")
+github.com/temple/tutorial
+github.com/temple/tutorial
+Generated project in /path/Documents/temple-tutorial
+```
+
+Now we can view the generated outputs of our project:
+
+```
+~/Documents/temple-tutorial ❯❯❯ tree
+.
+├── api
+│   └── dating-app.openapi.yaml
+├── auth
+│   ├── Dockerfile
+│   ├── auth.go
+│   ├── comm
+│   │   └── handler.go
+│   ├── config.json
+│   ├── dao
+│   │   ├── dao.go
+│   │   └── errors.go
+│   ├── go.mod
+│   ├── hook.go
+│   ├── metric
+│   │   └── metric.go
+│   ├── util
+│   │   └── util.go
+│   └── wait-for-kong.sh
+├── auth-db
+│   └── init.sql
+├── dating-app-user
+│   ├── Dockerfile
+│   ├── config.json
+│   ├── dao
+│   │   ├── dao.go
+│   │   └── errors.go
+│   ├── dating-app-user.go
+│   ├── go.mod
+│   ├── hook.go
+│   ├── metric
+│   │   └── metric.go
+│   └── util
+│       └── util.go
+├── dating-app-user-db
+│   └── init.sql
+├── dating-app.temple
+├── deploy.sh
+├── docker-compose.yml
+├── kong
+│   └── configure-kong.sh
+├── match
+│   ├── Dockerfile
+│   ├── comm
+│   │   └── handler.go
+│   ├── config.json
+│   ├── dao
+│   │   ├── dao.go
+│   │   └── errors.go
+│   ├── go.mod
+│   ├── hook.go
+│   ├── match.go
+│   ├── metric
+│   │   └── metric.go
+│   └── util
+│       └── util.go
+└── match-db
+    └── init.sql
+
+19 directories, 38 files
+```
+
+### Output Breakdown
+
+We'll break these outputs down to understand what each one means. 
+
+```
+/auth/
+/match/
+/dating-app-user/
```

Comment:
I’d be inclined to use an example that won’t be renamed, and only mention renaming as a side note at the end.